.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  run_tests:
+    name: tests
+    runs-on: ${{ matrix.os }}
```

Comment:
Do we need this indirection if the matrix only has one OS?

Reply:
We started doing it this way because:

1. It shows the OS version in use in the description for each check, since it's a matrix parameter.  This is handy for sanity checking that we didn't miss updating any tests during the last OS upgrade.
1. It simplifies the changes needed the next time we do need to test 2 different OS versions during an upgrade cycle.